tests/lib/rules/template-no-let-reference.js

```diff
@@ -56,15 +78,5 @@ ruleTester.run('template-no-let-reference', rule, {
       output: null,
       errors: [{ type: 'VarHead', message: rule.meta.messages['no-let'] }],
     },
-    {
-      code: `
```

Comment:
why remove this test?

Reply:
it's valid code.

_goofy code_, but valid.

Here is a demo: https://limber.glimdown.com/edit?c=IIAgJgpgtg9iMDMQHMaIJ4BoQCMCuALiAG4CGANgJZggDGMkAUIwAZvIBWAziFcRI3IQipEAF4QAckkBuRgB4C0AA7lSSgHyMQIABaVtIeaQ3yA9CYVmlUVeoha2LRkA&format=glimdown

Reply:
yes, but the rule is about referencing let,var vs const.
https://limber.glimdown.com/edit?c=IIAgJgpgtg9iMDMQHMaIJ4BoQCMCuALiAG4CGANgJZggDGMkAUIwAZvIBWAziFcRI3IQiADXQAvEAF4QAHgLQADuVIKAfKRy1ZAegVRlqiGoDcjeUpXrGIEAAtKNuWPFrdLteb2Wjnti0YgA&format=glimdown

Reply:
oh, interesting. it would ignore it if it is `xyz` instead of `Xyz`.
i think we just need to update the test then

Reply:
what's an invalid case?

Reply:
its invalid if a tag or mustache references a `let` or `var` variable, its valid if it references a `const`

Reply:
oh! I understand now. hm.